components/server/src/workspace/workspace-service.ts

```diff
@@ -437,10 +437,21 @@ export class WorkspaceService {
         user: User,
         workspaceId: string,
         options: StartWorkspaceOptions = {},
+        restrictToRegular = true,
```

Comment:
(nit) Should it be part of StartWorkspaceOptions?

Reply:
I hoped to get rid of this eventually, and that's why I tried to keep it separate. But maybe it should have gone in there anyway... :+1: